go-controller/pkg/ovn/service.go

```diff
@@ -339,3 +318,17 @@ func (ovn *Controller) svcQualifiesForReject(service *kapi.Service) bool {
 	_, ok := service.Annotations[OvnServiceIdledAt]
 	return !(config.Kubernetes.OVNEmptyLbEvents && ok)
 }
+
+func (ovn *Controller) isValidProtocol(proto kapi.Protocol) bool {
```

Comment:
oh, you already have this. but it should be in `util`

Reply:
ok I will move it into util. thanks.